src/transaction/boot_sr.c

```diff
@@ -5066,6 +5076,7 @@ boot_create_all_volumes (THREAD_ENTRY * thread_p, const BOOT_CLIENT_CREDENTIAL *
   /* Create double write buffer if not already created. DWB creation must be done before first volume.
    * DWB file is created on log_path.
    */
+  // TODO: !is_tran_server_with_remote_storage ?
```

Comment:
Not necessary to check here. When database is created for remote storage, no disk files will be created. This should not be reached.

Reply:
Ok